commands/netobserv

```diff
@@ -72,7 +72,11 @@ function packets() {
   esac
 }
 
-case "$1" in  
+required_yq_version="v0.0.0"
```

Comment:
@msherif1234 - qq , where do you set the required version here?

Reply:
its passed as an arg in the Makefile

Reply:
ah, I see it, the YQ_VERSION env var value already exists in main.